config/settings.yml

```diff
@@ -4,4 +4,9 @@ symws:
   headers: {}
 home_page_flash_message_html:
 redis:
-  url: <%= ENV.fetch 'REDIS_URL', 'redis://127.0.0.1:6379/1' %>
\ No newline at end of file
+  sidekiq:
```

Comment:
separate concerns, separate instances. that seems to be the going advice on Redis

https://github.com/mperham/sidekiq/wiki/Using-Redis#multiple-redis-instances

Reply:
👍